xds/src/main/java/io/grpc/xds/ClientXdsClient.java

```diff
@@ -1340,12 +1340,11 @@ private static boolean isTerminalFilter(FilterConfig filterConfig) {
           return StructOrError.fromStruct(RouteAction.forClusterSpecifierPlugin(
```

Comment:
I'm not sure if this [case](https://github.com/grpc/grpc-java/blob/012dbaf5be3fb0d532d977d288a0e42a58f30a7c/xds/src/main/java/io/grpc/xds/ClientXdsClient.java#L1548) be treated as ignore because of?
`3. If a cluster specifier references an existent plugin instance, but we don't support it (at this point, this implies it was optional), skip the Route` cc. @ejona86

Reply:
This will change, but is part of the RLS work and will change before RLS+xds is stabilized. I think it is fair to leave this as-is and change it as appropriate when we introduce is_optional for the cluster specifier plugins. We'll probably just need a new `if (pluginConfig == MISSING) return null;` case here.